tests/resources/codeflare-stack/mnist_ray_mini.ipynb

```diff
@@ -50,7 +50,8 @@
    },
    "outputs": [],
    "source": [
-    "cluster.wait_ready()"
+    "cluster.wait_ready()\n",
+    "sleep(30)"
```

Comment:
The reason for this is that the ray dashboard takes a few seconds to become accessible, so there is a gap between wait_ready and the job submission being usable. The short term solution is to simply add a small sleep before submitting the job, and the long-term solution (for next codeflare release) will likely involve waiting longer in wait_ready for the dashboard to be accessible, or adding better messaging in this edge case

We will create a follow-on issue for the wait_ready condition. Related, the notebook pods end up restarting almost immediately and then ending up in a crashloop at a different (earlier) point in the notebook. This makes it much harder to debug as the "useful" logs only showed up in the first instance of the crashloop -- every other instance fails due to an unrelated (but not critical) issue.

Thanks @MichaelClifford @Maxusmusti